news/_posts/core-weekly/2021-02-08-coreweekly-05-2021.md

```diff
@@ -0,0 +1,226 @@
+* [#22](https://github.com/PrestaShop/user-documentation-landing/pull/22): chore(deps-dev): bump eslint from 7.18.0 to 7.19.0. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+
+
+### Changes in developer documentation
+* [#853](https://github.com/PrestaShop/docs/pull/853): min version compat for tinymce config override. Thank you [@kpodemski](https://github.com/kpodemski)
+* [#624](https://github.com/PrestaShop/docs/pull/624): How to release PS, part 1, by [@matks](https://github.com/matks)
+
+
+### PrestaShop-modules
+* [#443](https://github.com/PrestaShop/PrestaShop-modules/pull/443): Remove 1.6 modules, by [@Progi1984](https://github.com/Progi1984)
+
+
+### Auto Upgrade module
+* [#378](https://github.com/PrestaShop/autoupgrade/pull/378): Fix phpstan tests, by [@atomiix](https://github.com/atomiix)
+* [#376](https://github.com/PrestaShop/autoupgrade/pull/376): Clear cache before enabling theme, by [@atomiix](https://github.com/atomiix)
+* [#374](https://github.com/PrestaShop/autoupgrade/pull/374): Fix error with private method and upgrade. Thank you [@nenes25](https://github.com/nenes25)
+
+
+### Wishlist block module
+* [#97](https://github.com/PrestaShop/blockwishlist/pull/97): Fix stats cache per shop id. Thank you [@pablopolyte](https://github.com/pablopolyte)
+
+
+### PrestonBot
+* [#113](https://github.com/PrestaShop/prestonbot/pull/113): Use Bux fix label instead of Bug, by [@atomiix](https://github.com/atomiix)
+
+
+### OnBoarding module
+* [#100](https://github.com/PrestaShop/welcome/pull/100): Bump eslint-config-prestashop from 0.0.2 to 0.1.0. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+
+
+### Example modules
+* [#41](https://github.com/PrestaShop/example-modules/pull/41): Fix composer autoloader configuration for demosymfonyform, by [@matks](https://github.com/matks)
+* [#35](https://github.com/PrestaShop/example-modules/pull/35): Rewrite demoextendsymfonyform1 module without CQRS, by [@matks](https://github.com/matks)
+
+
+### Google Analytics module
+* [#85](https://github.com/PrestaShop/ps_googleanalytics/pull/85): Update HookDisplayOrderConfirmation.php. Thank you [@Shoprunners](https://github.com/Shoprunners)
+
+
+### Customer data privacy block module
+* [#25](https://github.com/PrestaShop/ps_dataprivacy/pull/25): Release 2.0.1, by [@Progi1984](https://github.com/Progi1984)
+
+
+### Theme customization module
+* [#37](https://github.com/PrestaShop/ps_themecusto/pull/37): Release 1.2.1, by [@Progi1984](https://github.com/Progi1984)
+
+
+## Where to start contributing?
+
+What about [sending test emails in module ps_reminder](https://github.com/PrestaShop/PrestaShop/issues/18569)? This is a feature request submitted last year by long-time contributor [@PrestaShark](https://github.com/PrestaShark), and it is one of our [good first issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
```

Comment:
:warning: Warning, the ps_reminder module is going to be archived.

Reply:
Ah, thanks for the tip. I'm going to update the issue and change the Core Weekly.

Reply:
Fixed in https://github.com/PrestaShop/prestashop.github.io/pull/841/commits/460bda91878bce2eb3b4b56dbc2fa9b3e185cea6